pkg/workflow_execution.go

```diff
@@ -327,6 +348,49 @@ func (c *Client) CreateWorkflowExecution(namespace string, workflow *WorkflowExe
 	return workflow, nil
 }
 
+func (c *Client) AddWorkflowExecutionStatistic(namespace, name string, workflowTemplateID int64, createdAt time.Time, workflowOutcomeIsSuccess bool) (err error) {
+	tx, err := c.DB.Begin()
+	if err != nil {
+		return err
+	}
+	defer tx.Rollback()
+	if workflowOutcomeIsSuccess {
+		_, err := sb.Insert("workflow_executions").
+			SetMap(sq.Eq{
```

Comment:
Wonder if this can be done a bit more DRY with `sb`. Can't you just change only one key in the map?

Reply:
Hmm, let me see

Reply:
Was able to clean this up. feat/dev-core-144 (merging branch)